src/app/helptext/task-calendar/cloudsync/cloudsync-form.ts

```diff
@@ -31,6 +31,11 @@ export default {
     encryption_placeholder: T('Server Side Encryption'),
     encryption_tooltip: T('Choose <i>AES-256</i> or <i>None</i>.'),
 
+    fast_list_placeholder: T('Use --fast-list'),
+    fast_list_tooltip: T('Use fewer transactions in exchange for more RAM. This may also speed \
+    up or slow down your transfer. See [rclone documentation](https://rclone.org/docs/#fast-list) \
```

Comment:
s/your/the/

Reply:
The link label can be integrated with the first sentence:
`` fast_list_tooltip: T('[Use fewer transactions in exchange for more RAM.](https://rclone.org/docs/#fast-list) \
This can also speed up or slow down the transfer.'), ``